README.md

```diff
@@ -1 +1,28 @@
-# linda-portfolio-part1
\ No newline at end of file
+![](http://127.0.0.1:5500/Index.html)
+
+# linda-portfolio-part1
+
+I'm working on the portofolio project. This is the first part.  
+
+![screenshot](./Capture.png)
+
+## Built With
+
+-HTML
+-CSS
+-FIGMA
+
+### 👤 Author
+
+- GitHub: [@githubhandle](https://github.com/keza681)
+- Twitter: [@twitterhandle](https://twitter.com/LKeza19)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/linda-keza-a10150218/)
+
+#### Show your support
+
+Give a ⭐️ if you like this project!
+
+
+##### 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[**OPTIONAL**]
- [ ] I noticed your **MIT Licence** is a broken link. Please, be sure to make it as a link to your MIT LICENSE file after adding it to your root folder. Having a broken link in your README.md file is unprofessional.